src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/DiscoverVariantsFromContigAlignmentsSAMSpark.java

```diff
@@ -174,52 +170,118 @@ public static AlignedContig parseReadsAndOptionallySplitGappedAlignments(final I
+                                                   final DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection parameters,
                                                    final Broadcast<ReferenceMultiSource> broadcastReference,
-                                                   final String vcfOutputFileName, final Logger localLogger,
+                                                   final String vcfOutputFileName,
+                                                   final Logger localLogger,
                                                    final PairedStrandedIntervalTree<EvidenceTargetLink> evidenceTargetLinks,
                                                    final ReadMetadata metadata,
                                                    final SAMSequenceDictionary sequenceDictionary) {
 
         Utils.validate(! (evidenceTargetLinks != null && metadata == null),
                 "Must supply read metadata when incorporating evidence target links");
 
-        JavaRDD<VariantContext> annotatedVariants =
+        final JavaPairRDD<NovelAdjacencyReferenceLocations, Iterable<ChimericAlignment>> narlsAndSources =
                 alignedContigs.filter(alignedContig -> alignedContig.alignmentIntervals.size()>1)                                     // filter out any contigs that has less than two alignment records
                         .mapToPair(alignedContig -> new Tuple2<>(alignedContig.contigSequence,                                        // filter a contig's alignment and massage into ordered collection of chimeric alignments
                                 ChimericAlignment.parseOneContig(alignedContig, DEFAULT_MIN_ALIGNMENT_LENGTH)))
                         .flatMapToPair(DiscoverVariantsFromContigAlignmentsSAMSpark::discoverNovelAdjacencyFromChimericAlignments)    // a filter-passing contig's alignments may or may not produce novel adjacency
-                        .groupByKey()                                                                                                 // group the same novel adjacency produced by different contigs together
+                        .groupByKey();                                                                                                // group the same novel adjacency produced by different contigs together
+
+        narlsAndSources.cache();
+
+        if ( parameters.truthVCF != null ) {
+            final SVIntervalTree<String> trueBreakpoints =
+                    SVVCFReader.readBreakpointsFromTruthVCF(parameters.truthVCF, sequenceDictionary, parameters.truthIntervalPadding);
+
+            if ( assembledIntervals != null ) {
+                evaluateIntervalsAgainstTruth(assembledIntervals, trueBreakpoints, localLogger);
+            }
+
+            final SVIntervalTree<String> narlyBreakpoints =
+                    readBreakpointsFromNARLs(narlsAndSources.map(Tuple2::_1).collect(), sequenceDictionary, parameters.truthIntervalPadding);
```

Comment:
Real minor nitpick: in readBreakpointsFromNARLs and evaluateNARLsAgainstTruth you deviate from camel case, whereas everywhere else you treat "narl" as a camel case word.

Reply:
Fixed.